config/config.go

```diff
@@ -95,6 +95,7 @@ func setConfigDefaults(v *viper.Viper) {
 	v.SetDefault("request_limits.max_size_bytes", utils.REQUEST_MAX_SIZE_BYTES)
 	v.SetDefault("request_limits.max_num_values", utils.REQUEST_MAX_NUM_VALUES)
 	v.SetDefault("request_limits.max_ttl_seconds", utils.REQUEST_MAX_TTL_SECONDS)
+	v.SetDefault("request_limits.max_header_size_bytes", 0)
```

Comment:
Does 0 = no limit?

Reply:
If zero, the `http.Server` is created with a the `http` library's `DefaultMaxHeaderBytes` constant that equals 1 MB